src/SFA.DAS.ApplyService.InternalApi/appsettings.json

```diff
@@ -69,5 +69,27 @@
       "ExcludeSections": [ "4" ],
       "Roles": [ "GAC" ]
     }
+  ],
+  "CriminalComplianceGatewayConfig": [
```

Comment:
At some point we need to tackle this json file as opposed to adding to it.  Not one for now but definitely something to have a discussion about next time it needs changing.

Reply:
The criminal compliance check pages are all the same structure with only the page ID and question ID being looked up from QnA differing, so I thought it made sense to have this lookup data in config rather than in a case statement

Reply:
I agree, it does.  It's more the fact the file is 7k lines long that needs addressing. 😄